fission/function.go

```diff
@@ -204,11 +218,29 @@ func fnCreate(c *cli.Context) error {
 					ResourceVersion: pkgMetadata.ResourceVersion,
 				},
 			},
+			SecretList:     []fission.SecretReference{},
```

Comment:
I was wondering in what scenario will a function have more than one secrets or more than one configMaps.
It seems like we are not allowing CRUD on secrets and configMaps, instead, we are only allowing Create operation on them both. 
So, we may not need to have a list of secrets and list of configMaps. right?

Reply:
It is intentional. Though CLI only accept one secret/configmap. For advance user, they can use YAML to declare multiple secrets/configmaps (PR422). I will write down the constraint in the tutorial.